lib/bsdlib/nrf91_sockets.c

```diff
@@ -612,6 +612,9 @@ static inline int nrf91_socket_offload_poll(struct pollfd *fds, int nfds,
 		if (tmp[i].returned & NRF_POLLERR) {
 			fds[i].revents |= POLLERR;
 		}
+		if (tmp[i].returned & NRF_POLLNVAL) {
```

Comment:
Oh, just noticed, you actually have `NRF_POLLINVAL` in `nrf_socket.h`. Do you plan update it in `nrfxlib`?

Reply:
Yes, this is already in a PR for ```nrfxlib```.

Reply:
Ok, I got this PR merged, let's see what CI will say now.

Reply:
Do we need to push a button to start the CI for this PR again?

Reply:
I've triggered it, but it failed. Apparently, we need to update `west` manifest file to reference updated nrfxlib revision. PR is on it's way: #493.

I have to ask you to rebase once it's merged.